modules/k8s_cluster/main.tf

```
@@ -23,4 +23,9 @@ resource "helm_release" "helm_cloud_operator" {
     name  = "falco.enabled"
     value = var.enable_falco
   }
+
+  set {
+    name  = "httpsProxy"
```

Match the property value's name per https://github.com/illumio/cloud-operator/pull/195/files#diff-66181bd946d8466e14f946ae35b1437d91313ae026d79b33890aa9dbcdec7c30R92:
```suggestion
    name  = "env.httpsProxy"
```